test/spec/crud/unified/findOneAndUpdate-let-yml

```diff
@@ -0,0 +1,96 @@
+description: "findOneAndUpdate-let"
```

Comment:
Looks like the file name got mangled.  The [original](https://github.com/mongodb/specifications/blob/master/source/crud/tests/unified/findOneAndUpdate-let.yml) ends with `.yml` not `-yml`.

Reply:
Yup, good catch!